blueocean-rest/README.md

```diff
@@ -110,18 +110,17 @@ To avoid needing a crumb to POST data, the header `Content-Type: application/jso
 
 # Security
 
+NOTE: JWT is disabled by default for now. to enable JWT authentication use FEATURE_BLUEOCEAN_JWT_AUTHENTICATION=true system property.
```

Comment:
Can you document two things here:
- what happens when the flag is on
- what happens when the flag is off